docs/execution_providers/Nuphar-ExecutionProvider.md

```diff
@@ -63,6 +63,22 @@ create_shared.sh -c /path/to/jit/cache/NUPHAR_CACHE_VERSION [-m optional_model_f
 # run Nuphar inference again with cached JIT dll
 ```
 
+### NGEMM
+NGEMM (Nuphar GEMM) is an optimized low-precision GEMM implementation based on compiler techniques.
+Please refer to our paper for more details of NGEMM: ["NGEMM: Optimizing GEMM for Deep Learning via Compiler-based Techniques"](https://arxiv.org/abs/1910.00178).
+
+#### NGEMM Tiling / Permutation Configuration
+NGEMM has default tiling parameters, but users can overwrite them through environment variables:
+* NUPHAR_IGEMM_TILE_M / NUPHAR_IGEMM_TILE_N / NUPHAR_IGEMM_TILE_K
+
+    These 3 parameters are the tiling sizes for corresponded dimensions of GEMM ([M x K] x [K x N]).
+    Set them to different values will generate GEMM with different tiling sizes.
+
+* NUPHAR_IGEMM_PERMUTE
+
+    This enviornment variable is to control the loop permutation in GEMM.
+    Default is not apply any loop permutation. Other options are "inner/outer/all", corresponding to apply permutation for only inner tile loops / only outer loops / both inner and outer loops.
```

Comment:
typo: "Default is not" -> "The default is to not"

Reply:
More typo: "corresponding to apply permutation for" -> "to "referring to applying permutations to ..., respectively."

Reply:
Thanks.